### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by Micr0Bit**
Updated the Installation-Steps and make it easier to understand

### CODE REVIEW

**Comment by MustaphaTR** [comment]
I don't think this is right aproach.

First of all do we really need the pickup indicator, d2 didn't have such afaik.

Second, there should be a way to make game read the D2K content from D2K folder, there is no need to move the files to d2 folder.

**Comment by Micr0Bit** [comment]
@MustaphaTR : of course the ultimate goal is to be independent from d2k ... in fact , we should make a ticket just for that , listing all the steps to achieve that ... and im aware that a "pickup-indicator" didnt exist in the original 

Making the installation-guide better to understand and make d2 launchable in its current state has more priority then creating probably stale tickets for issues that will obviously get fixed during the work on d2
whoever stumbles on this mod should be able to install without any problems "right now"

**Comment by MustaphaTR** [comment]
I think the issue here is the */v2 subfolder, mod.yaml defines directly D2K Content folder, while they are actually found in v2 subfolder in a new installation of OpenRA (you may still have  them on main content folder if your installation is old). That's arguablly easier and cleary better solution for that. As i said game can read d2k files from d2k folder, no need to mess with unnecessary copies.

**Comment by MustaphaTR** [comment]
Looks good to me now, but may need a commits may need a squashing. :+1:

**Comment by Micr0Bit** [comment]
@MustaphaTR : i need to get into this again (git) ... i dont think i can squash via the browser

but we are not in a hurry :D

File: README.md [hunk 1]
```diff
@@ -1,8 +1,33 @@
...
+  ```
+  DUNE.PAK
+  VOC.PAK
+  ATRE.PAK
+  HARK.PAK
+  ORDOS.PAK
+  INTRO.PAK
+  FINALE.PAK
+  ```
+  * Paste those files into your Documents/OpenRA/Content/d2 folder (if the d2 folder doesnt exist , create it)
```
Comment (abcdefg30) [inline README.md]: The space between `exist` and the comma should be removed.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
+  * Download D2-Source (https://github.com/OpenRA/d2/archive/master.zip)
+  * Extract both files , rename the 'd2-master' folder to 'd2' and move it into the just extracted OpenRA-Source folder  
+  -> OpenRA-release-20161019/mods
+  * Assuming that you already installed OpenRA before from the Website (not from source) , you need to switch to its Directory and copy 3 files : 
+  ```
+    'OpenRA.Game.exe'
+    'Eluant.dll'
+    'OpenRA.Mods.Common.dll' (OpenRA/mods/common)
+  ```
+  * Paste the files into ' /OpenRA-release-20161019/mods/d2/OpenRA.Mods.D2/dependencies'
```
Comment (abcdefg30) [inline README.md]: If you're updating this, could you also remove the space in front of the first `/` here as well, please?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,29 @@
...
+   - `VOC.PAK`
+   - `ATRE.PAK`
+   - `HARK.PAK`
+   - `ORDOS.PAK`
+   - `INTRO.PAK`
+   - `FINALE.PAK`
+  * Paste those files into your `Documents/OpenRA/Content/d2 folder` (if the d2 folder doesnt exist, create it)
+  * Keep sure that you atleast once started the Dune2000 mod (thus downloaded to content for it)
+  
+  * Run `make all` in `OpenRA-release-20161019` directory via terminal for linux and mac. Or open `OpenRA.Mods.D2.sln` solution in Visual Studio and build project for windows
```
Comment (abcdefg30) [inline README.md]: You can also use the `make all` on windows. ^^

**Comment by abcdefg30** [comment]
Fixed the indentation and links.

**Pull request merged**